docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
+services: 
+    api:
+        build:
+            context: ./api
+        volumes:
+            - ./api:/usr/src/app
+            - /usr/src/app/node_modules
+        ports: 
+            - 5000:5000
+            - 5001:5001 
+        environment: 
+            PORT: 5000
+            NODE_ENV: development
+            DATBASE_USER: test
+            DATABASE_NAME: codeboxtv
+            DATABASE_PASSWORD: test
+            DATABASE_HOST: postgres
+    db:
+        image: postgres:13.0-alpine
+        volumes:
+            - postgres-data:/var/lib/postgresql/data
+        ports:
+            - 5432:5432
+        environment: 
+            POSTGRES_PASSWORD: test
+            POSTGRES_USER: test
+            POSTGRES_DB: codeboxtv
+            POSTGRES_HOST_AUTH_METHOD: trust
+volumes:
+    postgres-data:
```

Comment:
What is this doing?

Reply:
This basically allows for data to persist across multiple sessions of booting up the postgres container (For example, if you were to kill your container, it wouldn't lose data). This might be old-fashioned: https://www.digitalocean.com/community/questions/how-to-create-a-persistent-data-volume-for-postgres-database-container-within-a-docker-project